spec/src/main/asciidoc/chapters/language/expressions.adoc

```diff
@@ -75,25 +75,57 @@ An _identifier expression_, with syntax given by `id_expression`, is assigned th
 
 ==== Function calls
 
-A _function call_ is the name of a function recognized by the Jakarta Query implementation, followed by a parenthesized list of argument expressions, with syntax given by `function_expression`.
+A _function call_ is the name of a function recognized by the Jakarta Query implementation, followed by a parenthesized list of argument expressions, with syntax given by:
+
+- `function_expression` in the core language, or
+- `#functions_returning_strings#`, `#functions_returning_datetime#`, and `#functions_returning_numerics#` in the full grammar of the extended language.
+
+This specification defines the standard functions listed in the table below.
+
+NOTE: Functions highlighted in yellow belong to the extended language and are not required for an implementation of the core language.
+
+[cols="15,15,15,15,~"]
+|===
+| Function name | Parameters | Parameter types | Type | Semantics
+
+| `abs` | 1 | Any numeric type | Same as argument | Evaluates to the absolute value of the numeric value to which its argument evaluates.
+| `#sign#` | 1 | Numeric | Integer | Evaluates to the sign of the numeric value of its argument.
+| `#mod#` | 2 | Both integer | Integer | Evaluates to the remainder when its first integer argument is divided by its second integer argument.
```

Comment:
We should mention how negative numbers are handled.

Reply:
Yeah, I was wondering if that might be the case. Let's explicitly state behavior is undefined for negative so that user will realize that if they attempt it, they are relying on the database behavior, which is not portable.